RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTHcalVarProducerFromRecHit.cc

```diff
@@ -127,6 +142,7 @@ void EgammaHLTHcalVarProducerFromRecHit::fillDescriptions(edm::ConfigurationDesc
   desc.add<std::vector<double> >("etThresHB", {0, 0, 0, 0});
   desc.add<std::vector<double> >("eThresHE", {0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2});
   desc.add<std::vector<double> >("etThresHE", {0, 0, 0, 0, 0, 0, 0});
+  desc.add<bool>("usePFThresholdsFromDB", true);
```

Comment:
This makes sure the parameters is inserted if missing from a config, with value "true", so no need to add it explicitly in any cfi file for this plugin.